frontend/src/components/invoice/models/InvoiceModel.ts

```diff
@@ -102,7 +104,7 @@ export default class InvoiceModel implements IAttachment {
   /** Some weird stuff happening here that EditInvoice probably depends on */
   setClient(client: undefined | ClientModel): InvoiceModel {
     this.client = client as ClientModel;
-    this.date = getInvoiceDate(client);
+    this.date = getInvoiceDate(client, this.config, this.date);
```

Comment:
Als die config eerst niet meegegeven werd, dan is het misschien gewoon niet nodig?

Reply:
kan wel ja, alleen volgens de uitleg van steven zat de logica ook anders in elkaar? heb een samenvatting gemaakt op #283